utils/shared.js

```diff
@@ -1546,7 +1546,7 @@ const getTemplateForEmailLink = (
     continueUrl,
     preferredLanguage = fieldMapping.english.toString()
 ) => {
-    return preferredLanguage === fieldMapping.spanish.toString()
+    return preferredLanguage.toString() === fieldMapping.spanish.toString()
```

Comment:
No need to convert to strings if they both are numbers.

Reply:
Solved